docs/src/modules/components/ChartComponentsGrid.js

```diff
@@ -0,0 +1,95 @@
+import * as React from 'react';
+import { alpha } from '@mui/material/styles';
+import Card from '@mui/material/Card';
+import CardMedia from '@mui/material/CardMedia';
+import Grid from '@mui/material/Grid';
+import Typography from '@mui/material/Typography';
+import Link from 'docs/src/modules/components/Link';
+
+function components() {
+  return [
+    {
+      title: 'Bar Chart',
+      srcLight: '/static/x/component-illustrations/bar-light.png',
+      srcDark: '/static/x/component-illustrations/bar-dark.png',
+      href: '/x/react-charts/bar/',
```

Comment:
Oops https://next.mui.com/x/react-charts/bar/

Fixed in 1a131e0b48e674aae755d21e8f40c554d1f63a07.

Reply:
Thank you!